src/pages/VideoDetail/VideoDetail.page.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import VideoDetailCard from '../../components/VideoDetailCard';
+import { useVideoContext } from '../../store/VideoContext';
+
+function VideoDetailPage() {
+  const { state } = useVideoContext();
```

Comment:
**Tip**: You can get extra fancy with destructuring if you'd lke

`const { state: {currentVideo}} = useVideoContext(); `

and then you can use `currentVideo.videoThumbnail` below